vaadinfrontend/src/main/java/life/qbic/datamanager/ClientDetailsProvideImpl.java

```diff
@@ -0,0 +1,27 @@
+package life.qbic.datamanager;
+
+import com.vaadin.flow.component.page.ExtendedClientDetails;
+import com.vaadin.flow.component.page.Page.ExtendedClientDetailsReceiver;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import java.util.Optional;
+
+/**
+ * Receives client details from vaadin and provides them internally.
+ */
+@SpringComponent
+public class ClientDetailsProvideImpl implements ExtendedClientDetailsReceiver,
```

Comment:
Should be named "Provider" instead of "Provide", no?

Reply:
yes